inst/include/common/data_object.hpp

```diff
@@ -44,30 +44,42 @@ namespace fims {
  */
 template <typename Type>
 struct DataObject : public fims::FIMSObject<Type> {
-  static uint32_t id_g;
-  std::vector<Type> data;
-  size_t dimensions;
-  size_t imax;
-  size_t jmax;
-  size_t kmax;
-  size_t lmax;
+  static uint32_t id_g; /*!< id of the Data Object >*/
```

Comment:
class member documentation in FIMS uses two styles: /*!<>*/ and /**<>*/
recommendation: decide which style to use throughout and make changes during MQ

Reply:
Issue #282